command_line/show.py

```diff
@@ -26,6 +27,9 @@
 show_scan_varying = False
   .type = bool
   .help = "Whether or not to show the crystal at each scan point."
+show_experiment_has_model = False
```

Comment:
Perhaps this would be a better parameter (and relevant function) name?
```suggestion
show_model_connectivity = False
```